### DISCUSSION TITLE
Any plans for the book to be finished

### DISCUSSION THREAD

**Discussion by ZeyadMoustafaKamal** [rust-hosted-langs/book#62] [General]
The last commit was before like 4 months. The book is so useful not just to learn how to make a programming language but also to learn rust by building stuff. So I wonder about your plans for the next few chapters.

**Comment by pliniker** [comment]
Thank you, I am glad to hear the book is useful!

Life is busy but I am still working on the code and the book. Currently I am thinking through some aspects of garbage collection and how to best implement it.

**Reply by ZeyadMoustafaKamal** [reply]
I am waiting to hear more.

**Reply by pliniker** [reply]
There are good questions in this issue, which I am thinking about.
https://github.com/rust-hosted-langs/book/issues/60
